src/main/java/careneighbors/hospital/Hospital.java

```diff
@@ -19,35 +23,42 @@ public class Hospital {
 
     @NonNull
     @Column(nullable = false)
-    private String companyName;
+    private String companyName;//todo 병원명
 
     @NonNull
     @Column(nullable = false)
-    private String address;
+    private String address;//Todo 병원 주소
 
     @NonNull
     @Column(nullable = false)
-    private String contactNumber;
+    private String contactNumber;//Todo 병원 번호
 
     @NonNull
     @Column(nullable = false)
-    private String registrationNumber;
+    private String registrationNumber;//Todo 병원등록번호
 
     @NonNull
     @Column(nullable = false)
-    private String type;
+    private String type; //Todo 병원 사립 공립 유형
 
     @NonNull
     @Column(nullable = false)
-    private Integer bedCount;
+    private Integer bedCount; //Todo 병상 수
 
     @Column(nullable = true)
-    private String website;
+    private String website;//Todo 병원 홈페이지
 
     @Column(nullable = true)
-    private String imageUrl;
+    private String imageUrl; //Todo 병원 이미지
+
+    @OneToMany(mappedBy = "hospital")
+    private List<PatientHospital> patientsHospital; //Todo 병원환자 테이블 관계설정
```

Comment:
```suggestion
    private List<PatientHospital> patientsHospitals; //Todo 병원환자 테이블 관계설정
```
리스트타입 변수명은 명시적으로 s붙여주시는게 좋아요